WooCommerce/Classes/Model/ReviewAge.swift

```diff
@@ -25,31 +25,15 @@ extension ReviewAge {
     /// Returns the Age entity that best describes a given timespan.
     ///
     static func from(startDate: Date, toDate: Date) -> ReviewAge {
-        let components = [.day, .weekOfYear, .month] as Set<Calendar.Component>
-        let dateComponents = Calendar.current.dateComponents(components, from: startDate, to: toDate)
+        let timeDifference = toDate.timeIntervalSince(startDate)
+        let oneDayInSeconds: TimeInterval = 86_400
```

Comment:
By declaring 86400 seconds as one day explicitly, I see might be using the wrong numbers in the tests? I might be mistaken, but, for example:

- The comment states _"We are going to assume that days are 86400 milliseconds"_. Should be 86400.
- `testAgeCalculationsReturnLast24HoursAgeFor24HoursAnd1Second` should use 86400 + 1 second, uses 84461 instead.
- If `testAgeCalculationsReturnLast24HoursAgeFor23Hours` calculated 1 second before a day, should be 86400 - 1, but uses 84399 instead
- `testAgeCalculationsReturnLast24HoursAgeFor12Hours` uses 43200 correctly.
- `testAgeCalculationsReturnLast7DaysAgeForAlmostSevenFullDays` declares 7 days should be 604800, uses 604760 instead

Side note: Perhaps re-declaring these as 'let numberOfDays` and then, in the tests, passing `numberOfDays * oneDayInSeconds` would be more descriptive?

Reply:
Good point! I updated the tests in 8e2844edd290c8e28ca7282e983a72f593acf24b. I removed `testAgeCalculationsReturnLast24HoursAgeFor24HoursAnd1Second` because this case will now return "Last 7 days" - I don't see the point of updating the implementation to support this case. LMK what you think.